spec.emu

```diff
@@ -1705,36 +1741,20 @@
           <dl class="header"></dl>
           <emu-grammar>
             Scopes :
-              OriginalScopeTreeList `,` TopLevelItemList
-          </emu-grammar>
-          <emu-alg>
-            1. Perform DecodeScopesInfoItem of |OriginalScopeTreeList| with arguments _info_, _state_ and _names_.
-            1. Perform DecodeScopesInfoItem of |TopLevelItemList| with arguments _info_, _state_ and _names_.
-          </emu-alg>
-          <emu-grammar>
-            OriginalScopeTreeList :
-              OriginalScopeTreeList `,` OriginalScopeTreeItem
+              OriginalScopeTreeList
           </emu-grammar>
           <emu-alg>
-            1. Perform DecodeScopesInfoItem of |OriginalScopeTreeList| with arguments _info_, _state_ and _names_.
-            1. Perform DecodeScopesInfoItem of |OriginalScopeTreeItem| with arguments _info_, _state_ and _names_.
+            1. Let _originalScopes_ be the DecodedOriginalScopeTrees of |OriginalScopeTreeList| with arguments _state_ and _names_.
+            1. Set _info_.[[Scopes]] to _originalScopes_.
           </emu-alg>
           <emu-grammar>
-            OriginalScopeTreeItem :
-              OriginalScopeTree
-          </emu-grammar>
-          <emu-alg>
-            1. Set _state_.[[ScopeLine]] to 0.
-            1. Set _state_.[[ScopeColumn]] to 0.
```

Comment:
Did this get ported over? I was not able to find the resetting of the scope line/column state in-between top-level original scopes.